orders/views.py

```diff
@@ -1,3 +1,107 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http     import JsonResponse
+from django.views    import View
+
+from users.models    import Destination
+from products.models import Product, ProductOption
+from .models         import Order, Cart, Status
+from users.utils     import login_required
+from orders.utils    import refresh_order, query_debugger
+
+class CartView(View):
+    @query_debugger
+    @login_required
+    def post(self, request):
+        try:
+            data           = json.loads(request.body)
+            quantity       = data['quantity']
+            product        = Product.objects.get(id=data['product'])
+            user           = getattr(request,'user',None)
+            before_execute = Status.objects.get(id=1)
+            option         = product.productoption_set.get(id=data['option'])
```

Comment:
```suggestion
            data           = json.loads(request.body)
            quantity       = data['quantity']
            product_id        = data['product']
            user           = getattr(request,'user',None)
            before_execute = Status.objects.get(id=1)
            option_id         = data['option']
```
해당 객체 자체가 필요한 것이 아니라, 객체를 다른 클래스의 FK로 사용하실 것이라면, 꼭 객체 자체를 `get` 하지 않고 id만 쓰셔도 됩니다. 왜냐하면 객체를 `get` 할 때에는 SELECT 쿼리문이 발생하기 때문에 줄일 수 있다면 줄이는 것이 좋습니다